egs2/librimix/sot_asr1/conf/tuning/train_pit_asr_transformer_wavlm.yaml

```diff
@@ -0,0 +1,90 @@
+# network architecture
```

Comment:
PIT model may use different data preparation with SOT. It may be a bit surprising to put this config, especially this recipe folder is under `sot_asr1`. I don't know. Maybe we can explicitly make some notes about this config, e.g. what data preparation, what purpose, what differences, and how to use are about it.

Reply:
The motivation for including PIT  is to compare PIT and SOT on the same dataset (overlapped speech with a certain time delay). It is ok for me to remove it and only keep the SOT file here~